tests/unit/button/button_options.js

```diff
@@ -8,14 +8,14 @@ module( "button: options" );
 test( "disabled, explicit value", function() {
 	expect( 9 );
 
-	var element = $( "#radio01" ).button({ disabled: false });
+	var element = $( "#button" ).button({ disabled: false });
 	deepEqual( element.button( "option", "disabled" ), false, "disabled option set to false" );
```

Comment:
Now that you mention it, I remember struggling with that logic, so I agree that it makes sense to explicitly test that. Let's at least update the assertion message to capture that, though. And add tests that cover the other usecases, since the two existing both set explicit values, while we should also test for a disable property set in markup.